### PR TITLE
Added Colours: New icon colors added and modified typoPrimary and iconDefault

### PR DESCRIPTION
**Pull request by anjiattri**
## 📝 Summary

- added iconSecondary and iconTertiary
- typoPrimary change in dark theme
- iconDefault change in light theme

## 📌 Related Issue
NA

## ✅ Checklist
- [x] Code compiles and passes lint checks
- [ ] Added/Updated tests if relevant
- [ ] Updated documentation/comments where needed
- [ ] No console warnings or errors
- [x] Mobile and responsive behavior verified
- [x] Cross-browser compatibility checked
- [x] All reviewers added and relevant labels applied

## 🧪 How to Test (Optional)
Steps to verify this PR manually:

1. clone RDS
2. build RDS
3. replace dist to your project's node_modules -> @am92/rreact-design-system -> dist

## 🧩 Notes
Any extra context, edge cases, or known limitations.

### CODE REVIEW

File: src/Components/DsIcon/DsIcon.Overrides.ts [hunk 1]
```diff
@@ -9,78 +11,8 @@ export const DsIconOverrides = {
...
-      fontSizeFrostbite: {
-        fontSize: 'var(--ds-typo-fontSizeFrostbite)'
-      },
-      fontSizeBlizzard: {
-        fontSize: 'var(--ds-typo-fontSizeBlizzard)'
-      },
-      fontSizeIceAge: {
-        fontSize: 'var(--ds-typo-fontSizeIceAge)'
-      }
+      ...COLOR_STYLES,
```
Comment (ankitgandhi452) [inline src/Components/DsIcon/DsIcon.Overrides.ts]: Please move back to plain objects format
